learning/tour-of-beam/frontend/lib/models/content_tree.dart

```diff
@@ -16,13 +16,9 @@
  * limitations under the License.
  */
 
-import 'package:json_annotation/json_annotation.dart';
-
+import '../repositories/models/get_content_tree_response.dart';
 import 'module.dart';
 
-part 'content_tree.g.dart';
-
-@JsonSerializable(createToJson: false)
```

Comment:
`ModuleModel` is no longer deserialized with `json_serializable`, so this one is too.